src/Tribe/Attendees_Table.php

```diff
@@ -200,6 +200,37 @@ public function column_ticket( $item ) {
 	 * @return string
 	 */
 	public function column_check_in( $item ) {
+		$default_checkin_stati = array();
+		$provider              = $item['provider_slug'];
+		$order_id = $item['order_id'];
+
+		/**
+		 * Filters the order stati that will allow for a ticket to be checked in for all commerce providers.
+		 *
+		 * @since 4.1
+		 *
+		 * @param array  $default_checkin_stati An array of default order stati that will make a ticket eligible for check-in.
+		 * @param string $provider              The ticket provider slug.
+		 * @param int    $order_id              The order post ID.
+		 */
+		$check_in_stati = apply_filters( 'event_tickets_attendees_checkin_stati', $default_checkin_stati, $provider, $order_id );
+
+		/**
+		 * Filters the order stati that will allow for a ticket to be checked in for a specific commerce provider.
+		 *
+		 * @since 4.1
+		 *
+		 * @param array  $default_checkin_stati An array of default order stati that will make a ticket eligible for check-in.
+		 * @param int    $order_id              The order post ID.
+		 */
+		$check_in_stati = apply_filters( "event_tickets_attendees_{$provider}_checkin_stati", $default_checkin_stati, $order_id );
```

Comment:
Should this line pass `$check_in_stati` instead of `$default_checkin_stati`? As it stands, if someone adds a filter on `event_tickets_attendees_checkin_stati` the result will be lost when this line executes.